contracts/tests/test_channel_create.py

```diff
@@ -115,6 +117,56 @@ def test_channel_223_create_bounty_limit(get_block, owner, get_accounts, uraiden
     assert channel_data[1] == channel_deposit_bugbounty_limit
 
 
+def test_create_token_fallback_uint_conversion(
+    contract_params,
+    owner,
+    get_accounts,
+    uraiden_instance,
+    token_instance):
+    token = token_instance
+    (sender, receiver) = get_accounts(2)
+
+    # Make sure you have a fixture with a supply > 2 ** 192 + 100
+    deposit = contract_params['supply'] - 100
+    txdata = bytes.fromhex(receiver[2:].zfill(40))
+
+    # Fund accounts with tokens
+    token.transact({"from": owner}).transfer(sender, deposit)
+    assert token.call().balanceOf(sender) == deposit
+
+    # Open a channel with tokenFallback
+    if deposit > 2 ** 192:
+        with pytest.raises(tester.TransactionFailed):
+            txn_hash = token_instance.transact({"from": sender}).transfer(
+                uraiden_instance.address,
+                deposit,
+                txdata
+            )
+
+
+def test_channel_223_event(owner, get_accounts, uraiden_instance, token_instance, event_handler, print_gas):
+    token = token_instance
+    ev_handler = event_handler(uraiden_instance)
+    (sender, receiver) = get_accounts(2)
+    deposit = 1000
+    txdata = bytes.fromhex(receiver[2:].zfill(40))
+
+    # Fund accounts with tokens
+    token.transact({"from": owner}).transfer(sender, deposit + 100)
+
+    txn_hash = token_instance.transact({"from": sender}).transfer(uraiden_instance.address, 1000, txdata)
+
+    # Check creation event
+    ev_handler.add(
+        txn_hash,
+        uraiden_events['created'],
+        checkCreatedEvent(sender, receiver, deposit)
+    )
+    ev_handler.check()
+
+    print_gas(txn_hash, 'channel_223_create')
```

Comment:
the print gas is for test debugging purposes?

Reply:
Yes, quick way to have some values after testing.